tests/trading/test_cancelOrder.py

```diff
@@ -116,9 +125,9 @@ def test_exceptions(contractsFixture):
 
     # cancelOrder exceptions
     with raises(TransactionFailed):
-        cancelOrder.publicCancelOrder(0, orderType, market.address, outcomeID, sender=tester.k1)
+        cancelOrder.publicCancelOrder(ZEROED_ORDER_ID, orderType, market.address, outcomeID, sender=tester.k1)
     with raises(TransactionFailed):
-        cancelOrder.publicCancelOrder(orderID + 1, orderType, market.address, outcomeID, sender=tester.k1)
+        cancelOrder.publicCancelOrder('\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01', orderType, market.address, outcomeID, sender=tester.k1)
```

Comment:
You can simplify `'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01'` to `'\x01'.rjust(32, '\x00')`.